### DISCUSSION TITLE
How to send multiple attachment?

### DISCUSSION THREAD

**Discussion by jasonho1308** [AsamK/signal-cli#474] [Q&A]
I am writing a program to get email message and attachment and sent it to signal, in some email, I got multiple attachment, I want to send all of it to signal but fail, here is my attempt, what did I do wrong?

```
signal-cli -u username send -m message -g group_id -a path/attachment1 path/attachment2
signal-cli -u username send -m message -g group_id -a 'path/attachment1' 'path/attachment2'
signal-cli -u username send -m message -g group_id -a "path/attachment1" "path/attachment2"
signal-cli -u username send -m message -g group_id -a [path/attachment1 [path/attachment2]]
signal-cli -u username send -m message -g group_id -a [[path/attachment1] [path/attachment2]]
```
All so this attempt return (no such file or directory)
I try this code also:
```
signal-cli -u username send -m message -g group_id -a path/attachment1
```
It works fine, what did I do wrong and how to fix it? Thanks

**Comment by jasonho1308** [comment]
Problem fix, it is my problem

**Reply by GabrielBozza** [reply]
Can you show what you did to make it work? I'm having the same problem and can't figure out how to solve it.

**Reply by Costly** [reply]
Now i'm having the same problem and can't figure it out too. Anyone knows how to send multiple files in one message?

**Reply by jasonho1308** [reply]
Sorry for giving such a poor answer in the past ☹

The project that I am working on, using this cli, has ended long time ago, after reviewing the code I wrote before, the solution I used to this problem is to send the attachments one by one per command, it seems it is impossible to send multiple files in one message using the cli (even on [mobile](https://community.signalusers.org/t/sending-multiple-attachments-at-once/2459) I believe)

Therefore, the solution is call the command multiple times to send multiple files. Hope this helps
